src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -83,6 +83,8 @@
 				@mouseleave="showReloadButton = false"
 				@blur="showReloadButton = false">
 				<NcButton v-if="sendingErrorCanRetry && showReloadButton"
+					type="tertiary"
+					size="small"
```

Comment:
Its height has extra px to the message height

https://github.com/user-attachments/assets/0dc8d555-d714-482a-985f-4a9d40c5a935